### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by jburel**
Bump version to 5.2.7

### CODE REVIEW

**Comment by jburel** [comment]
@sbesson travis is green

File: pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <groupId>ome</groupId>
   <artifactId>pom-omero-client</artifactId>
   <packaging>pom</packaging>
-  <version>5.2.12-SNAPSHOT</version>
+  <version>5.2.13-SNAPSHOT</version>
```
Comment (sbesson) [inline pom.xml]: Should this be 5.2.12 ?

Comment (jburel) [inline pom.xml]: now that we have snapshot
can probably keep it like that

**Comment by jburel** [comment]
@sbesson: it should be sorted.

**Pull request merged**